core/index.ts

```diff
@@ -64,7 +65,7 @@ export default class Core implements ICore {
   }
 
   public async goto(url: string) {
-    return this.tab.runCommand<void>('goto', url);
+    return this.tab.runCommand<IResourceMeta>('goto', url);
```

Comment:
Is there really value in returning the resource?

Reply:
This code change is simply correcting the typing, not changing the return type.

I do believe there's value if you are loading something other than an html page - ie, you're loading a json file.